### PR TITLE
Project cleanup

### PR DESCRIPTION
**Pull request by xaviarias**

### CODE REVIEW

**Review by SweeXordious — APPROVED** [review]

**Comment by xaviarias** [comment]
@SweeXordious I think this should wait until your PR is merged:

```
Caused by: org.gradle.internal.exceptions.LocationAwareException: Task 'shadowJar' not found in root project 'Test'.
```

**Comment by SweeXordious** [comment]
@xaviarias no worries. Just merge and I will take care of the rest

**Pull request merged**
